source/extensions/filters/network/dubbo_proxy/deserializer.h

```diff
@@ -1,23 +1,52 @@
 #pragma once
 
+#include <string>
+#include <unordered_map>
+
 #include "envoy/buffer/buffer.h"
 
+#include "common/common/assert.h"
+#include "common/config/utility.h"
 #include "common/singleton/const_singleton.h"
 
 namespace Envoy {
 namespace Extensions {
 namespace NetworkFilters {
 namespace DubboProxy {
 
+enum class DeserializerType : uint8_t {
```

Comment:
I think there should be a reuse `SerializationType`

https://github.com/envoyproxy/envoy/blob/master/source/extensions/filters/network/dubbo_proxy/protocol.h#L53

Reply:
I have modified it.